File: Task-03/Java/subtask2.java
import java.io.FileReader;
import java.io.FileWriter;
import java.io.IOException;
 
class subtask2 {
 
    public static void main(String[] args)
    {

        try {

            FileReader fr = new FileReader("input.txt");
            FileWriter fw = new FileWriter("output.txt");
            String str = "";
            int i;
            while ((i = fr.read()) != -1) {
                str += (char)i;
            }
            System.out.println("Content of the file:");
            System.out.println(str);
            fw.write(str);
            fr.close();
            fw.close();
 
            // Display message
            System.out.println(
                "File reading and writing both done");
        }
 
        catch (IOException e) {
            System.out.println(
                "There are some IOException");
        }
    }
}

File: Task-03/Rust/subtask2.rs
use std::fs::File;
use std::io::{self, Read, Write};

fn main() -> io::Result<()> {
    let mut input_file = File::open("input.txt")?;        //reads the file

    let mut content = String::new();                     //store the content of file
    input_file.read_to_string(&mut content)?;

    let mut output_file = File::create("output.txt")?; 

    output_file.write_all(content.as_bytes())?;               //writes the content

    Ok(())
}
